### DISCUSSION TITLE
pie menu improvement

### DISCUSSION THREAD

**Discussion by etma101** [JulienHeijmans/quicksnap#32] [Ideas]
i don't know how this affects performance but in the pie menu being able to choose multiple snapping types using shift+click (for example vertices and faces) would improve the process flow and reduce the frequency of using pie menu

**Comment by JulienHeijmans** [comment]
Hey, so I don't plan to allow that specifically, but I will add somethng that should really help with the " frequency of using pie menu" issue.
I am simply adding some hotkeys: 1,2,3 will allow to quickly change the snap target: verts, mid-edges, face centers

https://user-images.githubusercontent.com/35562774/197704036-f72cb609-c63c-426e-9161-e691f82332d0.mp4

I will also add a small icon next to the cursor, that shows the current snap target.

**Reply by etma101** [reply]
oh i never thougt of that, this is a lot better and intuitive. thanks

**Reply by etma101** [reply]
just a question: will hotkeys work during moving? (for changing target type)

**Reply by JulienHeijmans** [reply]
Yes, it's how they are supposed to be used. If you use the hotkey when picking the origin, it changes the snap type of the origin, if you use the hotkey during moving, it will change the snap type of the target.
